model/task_lifecycle.go

```diff
@@ -1371,18 +1371,19 @@ func UpdateBuildAndVersionStatusForTask(t *task.Task) error {
 		return errors.Wrapf(err, "updating version '%s' status", taskVersion.Id)
 	}
 
-	if evergreen.IsPatchRequester(taskVersion.Requester) {
+	if evergreen.IsPatchRequester(taskVersion.Requester) || evergreen.IsGitHubPatchRequester(taskVersion.Requester) {
```

Comment:
IsPatchRequester already calls IsGitHubPatchRequester so this may not be the right change

Reply:
:see_no_evil:  thank you! correct. 

After further investigation I closed the ticket and added some more information in a comment in the ticket.